sound/soc/sof/intel/hda-ctrl.c

```diff
@@ -285,7 +290,11 @@ int hda_dsp_ctrl_init_chip(struct snd_sof_dev *sdev, bool full_reset)
 
 	bus->chip_init = true;
 
+err:
 	hda_dsp_ctrl_misc_clock_gating(sdev, true);
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_HDA)
```

Comment:
should err: be line 295?

Reply:
@plbossart so this was indeed on purpose here, but let me rework this to make this clearer